CMakeLists.txt

```diff
@@ -192,6 +191,7 @@ if (NOT HIPIFY_CLANG_TESTS_ONLY)
   set(HIPIFY_LIBEXEC_INSTALL_PATH "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBEXECDIR}/hipify")
 
   install(
+    PROGRAMS ${CMAKE_CURRENT_SOURCE_DIR}/bin/hipify-perl
     PROGRAMS ${CMAKE_CURRENT_SOURCE_DIR}/bin/findcode.sh
     PROGRAMS ${CMAKE_CURRENT_SOURCE_DIR}/bin/findcode_headers.sh
```

Comment:
findcode.sh, findcode_headers.sh, ... are excluded in the old code. do we need to install it now?

Reply:
AFAIK they are still being shipped in packages, but even if that is not so now, their removal is not the point of this PR.